### ISSUE TITLE
What about geshi-1.2 ?

### ISSUE THREAD

**Issue by dcorb** [GeSHi/geshi-1.1#2]
What is the status of the development of Geshi?
Geshi 1.1 here in github
Geshi 1.0 here http://qbnz.com/highlighter/
References about Geshi 1.2 in the bottom: http://geshi.org

Thank you for migrating the repo to Github, definitely the right step. I would love to see more activity around Geshi.

**Comment by dcorb** [comment]
I found something in the doc/index.html of this repo:
https://github.com/GeSHi/geshi-1.1/blob/master/docs/index.html

```
<p>If you're reading this, thanks for helping test GeSHi 1.1! This is the brand-spanking new
version of GeSHi set to debut as GeSHi 1.2 whenever it's finished, and hopefully with your
help that time will be sooner rather than later. Perhaps before Duke Nukem Forever goes Gold
;)</p>
```

**Comment by apjanke** [comment]
Yeah. It uses Linux kernel version numbering conventions. Odd-numbered "1.x" series are the development versions that will turn into the subsequent even-numbered "1.y" stable series.
